packages/opencensus-instrumentation-grpc/src/grpc.ts

```diff
@@ -231,12 +244,10 @@ export class GrpcPlugin extends classes.BasePlugin {
   /**
    * Handler Server Stream and Bidirectional Stream Calls
    */
-  private serverStreamAndBidiHandler(
+  private serverStreamAndBidiHandler<RequestType, ResponseType>(
       plugin: GrpcPlugin, rootSpan: types.RootSpan, call: ServerCallWithMeta,
-      // tslint:disable-next-line:no-any
-      original: grpcModule.handleCall<any, any>,
-      // tslint:disable-next-line:no-any
-      self: grpcModule.handleCall<any, any>) {
+      original: grpcTypes.handleCall<RequestType, ResponseType>,
+      self: grpcTypes.handleCall<RequestType, ResponseType>) {
```

Comment:
I think `self: {}` would probably be sufficient here. It doesn't make sense for `original` and `self` to be the same type.

Reply:
Done!